akka-docs/src/main/paradox/persistence-query.md

```diff
@@ -303,10 +303,10 @@ Java
 And the `eventsByTag` could be backed by such an Actor for example:
 
 Scala
-:  @@snip [MyEventsByTagPublisher.scala](/akka-docs/src/test/scala/docs/persistence/query/MyEventsByTagPublisher.scala) { #events-by-tag-publisher }
+:  @@snip [MyEventsByTagSource.scala](/akka-docs/src/test/scala/docs/persistence/query/MyEventsByTagSource.scala) { #events-by-tag-publisher }
```

Comment:
So now it is not "such an Actor" any more

Reply:
Indeed!